examples/news_alerts/service.bal

```diff
@@ -0,0 +1,65 @@
+import ballerina/uuid;
+import ballerina/graphql;
+
+configurable boolean ENABLE_GRAPHIQL = ?;
+
+@graphql:ServiceConfig {
+    graphiql: {
+        enabled: ENABLE_GRAPHIQL
+    }
+}
+service /news on new graphql:Listener(9090) {
```

Comment:
Should we add some API documentation for each resource function. Just like in `snowtooth`.